test/linters/eslint.json

```diff
@@ -24,7 +24,7 @@
     "accessor-pairs": "error",
     "arrow-spacing": ["error", { "before": true, "after": true }],
     "block-spacing": ["error", "always"],
-    "brace-style": ["error", "1tbs", { "allowSingleLine": true }],
+    "brace-style": "off",
```

Comment:
why?

Reply:
I had an issue with the template injection. After the `try/catch` block I had to disconnect kuzzle in a `finally` block.
All snippets file ends with a newline character added automatically by most IDE and after injection the snippet looked like this:
```js
try {
  
} catch (error) {
  
}
finally {
  
}
```
But I figured out a way to avoid editing our linting configuration, I remove the last character of the snippet file if this is a `\n`.